src/python_routing_v02/troute/README.md

```diff
@@ -0,0 +1,11 @@
+## Notes:
+Cython doesn't understand namespace packages, or if it does, I don't know how to make it use them.  To work around that, in the namespace directory, add `__init__.pxd`.  Python will ignore this and can treat the directory as a namespace package, and Cython will see it as a "normal" package and allow relative imports to work correctly.
+
+
+Framework and Routing are too dependent to really use two differnt pacakges, even under the same namespace.
+
+It would make sense to refactor the framework so that routing is a subpackage.  Then perhaps implement a subpackage which allows plugging in different network types and routing types?  Some mix of namespace package (for the framework) with plugin like utility.
```

Comment:
@hellkite500 
I might be missing the point but I think this chunk of code (which is not yet merged, but will be soon, hopefully --- we need to decide how best to juggle it with the present PR) describes the front end unit of network preprocessing: 
https://github.com/jameshalgren/t-route/blob/self_loop_merge2/src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py#L1459-L1474

Basically, that creates a topological ordering that could be saved as a static object and be route-able, if the required dynamic inputs (lateral inflows, temporal and spatial boundary conditions, data assimilation targets) are supplied. Perhaps that can guide the continuing effort to streamline this as a generalized package.

Continuing the review...